src/views/evse.pug

```diff
@@ -37,79 +44,65 @@ html
-              #bookingNumberHelp.form-text Your booking number or reservation number.
+              label.form-label(for="inputBookingNumber")= __('BOOKINGNUMBER')
+              input#inputBookingNumber.form-control(name="bookingNumber" v-model="bookingNumber" aria-describedby="bookingNumberHelp" minlength="6")
+              #bookingNumberHelp.form-text= __('BOOKINGNUMBERHINT')
 
             .mb-3.form-check
-              input#tocCheck.form-check-input(type="checkbox" name="acceptedConditions" v-model="acceptedConditions" required="true")
+              input#tocCheck.form-check-input(type="checkbox" name="acceptedConditions" v-model="acceptedConditions")
               label.form-check-label(for="tocCheck")
-                | I accept the
+                | #{ __('IACCEPT') }
                 |
-                a(href=tenant.termsAndConditionsUrl target="_blank") Terms and Conditions
+                a(href=tenant.termsAndConditionsUrl target="_blank")= __('TERMSANDCONDITIONS')
                 |
-                | and the
+                | #{ __('ANDTHE') }
                 |
-                a(href=tenant.dataPrivacyUrl target="_blank") Data Privacy Agreement
+                a(href=tenant.dataPrivacyUrl target="_blank")= __('PRIVACYPOLICY')
                 | .
 
-            button.btn.btn-primary(type="submit") Start Charging
+            button.btn.btn-primary(type="submit")= __('STARTCHARGING')
 
         section(v-if="activeSection === 'charging'")
           form(action="/evseId/" + poi.evseid + "/stop" method="POST" @submit.prevent="submitStopForm")
             .alert.alert-success(role="alert")
               i.bi.bi-check-circle
               p
-                |
-                | Charging session
-                |
-                strong {{ chargingSession.id }}
-                | 
-                | started on behalf of booking number
-                |
-                strong {{ bookingNumber }}
-                | .
+                | #{ __('CHARGINGSTARTED') }
                 br
                 br
-                | A reservation of €50 has been added to your room.
-                | Once the charging has finished, the correct amount will be applied.
+                | #{ __('CHARGEDROOMONSTART') }
 
-            button.btn.btn-primary(type="submit") Stop Charging
+            button.btn.btn-primary(type="submit")= __('STOPCHARGING')
 
         section(v-if="activeSection === 'receipt'")
           form(@submit="submitCloseForm")
             .alert.alert-info(role="alert")
               i.bi.bi-receipt-cutoff
               p
-                |
-                | Charging session
-                |
-                strong {{ chargingSession.id }}
-                | 
-                | ended after {{ chargingSession.duration }} minutes on behalf of booking number
-                |
-                strong {{ bookingNumber }}
-                | .
+                | #{ __('CHARGINGENDED', { duration: '27' }) }
```

Comment:
Why is it hardcoded here instead of using the charging session?
```suggestion
                | #{ __('CHARGINGENDED', { duration: chargingSession.duration }) }
```